content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -111,17 +119,106 @@ $ kubectl apply -f @samples/sleep/notsleep.yaml@
 
 Note: `sleep` and `notsleep` are two simple applications that can serve as curl clients.
 
-Connect `productpage` to the Istio ingress gateway so you can access the bookinfo
-app from outside of the cluster:
+{{< tabset category-name="config-api" >}}
+
+{{< tab name="Istio classic" category-value="istio-classic" >}}
+
+Create an Istio [Gateway](/docs/reference/config/networking/gateway/) and
+[VirtualService](/docs/reference/config/networking/virtual-service/), so you can access the bookinfo app through the
+Istio ingress gateway:
 
 {{< text bash >}}
 $ kubectl apply -f @samples/bookinfo/networking/bookinfo-gateway.yaml@
 {{< /text >}}
 
-Test your bookinfo application, it should work with or without the gateway. Note: you can replace `istio-ingressgateway.istio-system` below with its load balancer IP (or hostname) if it has one:
+Set the environment variables for the Istio ingress gateway:
+
+{{< text bash >}}
+$ export GATEWAY_HOST=istio-ingressgateway
+$ export GATEWAY_SERVICE_ACCOUNT="$GATEWAY_HOST"-service-account
```

Comment:
This isn't really the hostname (i.e., without the .namespace). Since we are planning to move the k8s gateway to the default ns in the future, you should really include the ns in the env vars anyway.

```suggestion
$ export GATEWAY_HOST=istio-ingressgateway.istio-system
$ export GATEWAY_SERVICE_ACCOUNT=ns/istio-system/sa/istio-ingressgateway-service-account
```